### DISCUSSION TITLE
I use error reporting in widgetkit

### DISCUSSION THREAD

**Discussion by Jdb156158** [airbnb/lottie-ios#2403] [General]
The usage code is as follows：
LottieView(animation: .named("love"))
                  .playing()

error tips:
The body of the Widget entries' view contains the following unsupported types: PlatformViewRepresentableAdaptor<SwiftUIView<LottieAnimationView, ()>>,

Desktop widget display effect：
![image](https://github.com/airbnb/lottie-ios/assets/18107143/f6566e4a-4610-4e1f-a683-e90b51938c4b)

**Comment by Jdb156158** [comment]
有大佬解决了的吗，好像之前有人解决过

**Comment by tangtiancheng** [comment]
我也碰到了,作者大佬能解决下吗.怎么在小组件中使用
